omod-1.10/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_10/DrugReferenceMapController1_10Test.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.controller.openmrs1_10;
+
+
+import java.util.Set;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.Drug;
+import org.openmrs.DrugReferenceMap;
+import org.openmrs.api.ConceptService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.test.Util;
+import org.openmrs.module.webservices.rest.web.RestTestConstants1_10;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceControllerTest;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+public class DrugReferenceMapController1_10Test extends MainResourceControllerTest{
+	
+	private ConceptService conceptService;
+	
+    String term_uuid = "ee7693a0-8a55-22e3-8507-1f26fdd6e3f3";
+    
+    String mapType_uuid = "sece2323fa-0fa0-461f-9b59-6765997d84";
+    
+    String mapType_uuid2 = "8e2323fa-0fa0-461f-9b59-6765997d84td";
+    
+    String drug_uuid = RestTestConstants1_10.DRUG_REFERENCE_DRUG_UUID;
+    
+    String drugMap_uuid = RestTestConstants1_10.DRUG_REFERENCE_MAP_UUID;
+	
+	  String json = "{\n" + 
+	    		"  \"conceptReferenceTerm\":\"ee7693a0-8a55-22e3-8507-1f26fdd6e3f3\"\n" + 
+	    		",\n" + 
+	    		"  \"conceptMapType\": \"sece2323fa-0fa0-461f-9b59-6765997d84\"\n" + 
+	    		",\n" + 
+	    		"  \"drug\": \"8e2323fa-0fa0-461f-9b59-6765997d849d\"\n" + 
+	    		"}";
+	@Before
+	public void before() throws Exception {
+		conceptService = Context.getConceptService();
+		executeDataSet(RestTestConstants1_10.DRUG_REFERENCE_MAP_TEST_DATASET);
+	}
+	
+	@Override
+	public String getURI() {
+		
+		return "/drugreferencemap";
+	}
+
+	@Override
+	public String getUuid() {
+	
+		return RestTestConstants1_10.DRUG_REFERENCE_MAP_UUID;
+	}
+
+	@Override
+	public long getAllCount() {
+		
+		return 0;
+	}
+
+	@Test
+	public void ShouldCreateNewDruReferenceMap() throws Exception {
```

Comment:
Did you take a look at our method naming conventions?

Reply:
i corrected it